playbooks/reboot.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Reboot nodes if reboot is required
+  hosts: all
+  strategy: linear
```

Comment:
As I read docs linear does not imply that nodes are rebooted one at the time, but rather in a batches defined by fork limit. The default fork limit is 5, but this can be tweaked.  https://docs.ansible.com/ansible/latest/collections/ansible/builtin/linear_strategy.html#linear-strategy
I took other playbooks(all) as an example, where we specify linear strategy. I thought we would like to be explicit with strategy. 

I don't think the order of reboots matter and this could be change to free strategy, but imho reading logs with free strategy is more difficult to follow. This role doesn't have much task, so it might not be that big issue.

Reply:
I don't think we should use the free strategy. I'd prefer to stick with linear unless there is a compelling reason to use free. As you say, linear is the ansible default, it would be better to use the default. Linear does not turn parallel tasks into sequential, so I don't follow that reasoning.

If we wanted to reboot all nodes in parallel, regardless of the forks setting, that would need to be done with the async task pattern on the single task to reboot the nodes, which would need to be used for linear or free regardless.

Reply:
Can we move forward with this? @jpodivin I don't think it's necessary to block this on the strategy discussion.